### PR TITLE
Fix doc generation

### PR DESCRIPTION
**Pull request by okurz**
* Make sure a valid up-to-date license in all files
* Fix documentation publishing on gh-pages

### CODE REVIEW

**Comment by okurz** [comment]
this seems to have worked to update index but I'm not sure about https://github.com/os-autoinst/openQA/blob/gh-pages/docs/api/testapi.html

File: script/generate-documentation [hunk 1]
```diff
@@ -35,7 +34,7 @@ function update_docs {
 
         cd ..
         git add _includes/api.html
-        git add index.html current.pdf
+        git add docs/index.html docs/current.pdf
```
Comment (foursixnine) [inline script/generate-documentation]: Before it was a find for all the files, so you also need to add docs/api/testapi.html

Comment (okurz) [inline script/generate-documentation]: I see. But why did you delete the find in the first place?

Comment (okurz) [inline script/generate-documentation]: https://github.com/os-autoinst/openQA/pull/1347
